.github/workflows/gh-docker-test.yaml

```diff
@@ -0,0 +1,55 @@
+name: GitHub Docker Test
+env:
+  GOOGLE_APPLICATION_CREDENTIALS: /tmp/jade-dev-account.json
+  GOOGLE_CLOUD_PROJECT: broad-jade-integration
+  GOOGLE_REGION: us-central1
+  K8_CLUSTER: integration-master
+on:
+  workflow_dispatch: {}
+  pull_request:
+    branches:
+      - fb-dr-1528-gh-docker
+jobs:
+  gh_docker_test:
+    timeout-minutes: 180
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Import Vault Secrets for Development GCR
+        uses: hashicorp/vault-action@v2.1.2
+        with:
+          url: ${{ secrets.VAULT_ADDR }}
+          method: approle
+          roleId: ${{ secrets.ROLE_ID }}
+          secretId: ${{ secrets.SECRET_ID }}
+          secrets: |
+            secret/dsde/datarepo/dev/sa-key-b64 sa | B64_APPLICATION_CREDENTIALS ;
+      - name: Decode Vault token
+        run: |
+          base64 --decode <<< ${B64_APPLICATION_CREDENTIALS} > ${GOOGLE_APPLICATION_CREDENTIALS}
+      - name: Install Google Cloud SDK
+        uses: google-github-actions/setup-gcloud@v0.2.1
+        with:
+          project_id: ${{ env.GOOGLE_CLOUD_PROJECT }}
+          service_account_key: ${{ env.B64_APPLICATION_CREDENTIALS }}
+          export_default_credentials: true
+      - name: Configure Google Cloud SDK
+        uses: broadinstitute/datarepo-actions/actions/configure-gcloud-sdk@fb-dr-1673-configure-gcloud-sdk
+      - name: Add Runner IP to allowlist
+        uses: broadinstitute/datarepo-actions/actions/add-runner-ip@fb-dr-1688-add-runner-ip
```

Comment:
As per my comments elsewhere, I think the remove/add actions could be subject to a race condition, if two different actions are trying to add/remove at the same time. Not sure it's worth setting up a lock to handle it, but it's worth knowing about in case things fail in a weird flaky way.

Reply:
Oh, interesting. So, we had to add a retry loop for this step b/c it was flaky. That might explain it.

Reply:
> Oh, interesting. So, we had to add a retry loop for this step b/c it was flaky. That might explain it.

I'm not worried about getting an error due to steps overlapping. The problem is that updating the allowlist or updating the IAM policy is a two step operation – fetch current data; post edited version of current data – so if two actions are both updating the data at the same time, you could see fetch1, fetch2, post1, post2. If that occurs, the data in post1 will be overwritten with post2.

Reply:
Yes, there is a race condition here that is a challenge to resolve. I am not sure what the best mechanism is to resolve this, but I added a comment here: https://github.com/broadinstitute/datarepo-actions/pull/50